src/DataValues/QuantityValue.php

```diff
@@ -91,155 +70,59 @@ public function __construct( DecimalValue $amount, $unit, DecimalValue $upperBou
 	 * @note: if the amount or a bound is given as a string, the string must conform
 	 * to the rules defined by @see DecimalValue.
 	 *
-	 * @since 0.1
+	 * @since 0.7
```

Comment:
As with all `@since` tags in this file, this should not change because the method signature did not changed, as far as I can see. That was the whole point of renaming the two classes, wasn't it?